draft-packages/modal/KaizenDraft/Modal/Primitives/GenericModal.tsx

```diff
@@ -289,4 +289,28 @@ const createAriaHider = (dialogNode: HTMLElement): (() => void) => {
   }
 }
 
-export default GenericModalContainer
+const ClientOnly = ({
```

Comment:
Would you see this wrapper being used with more components in the future? If so, it might be worth to at least extract it out to its own file (even if it stays in the same package for now).

Would you be able to add a test for this new functionality as well, please.